x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_actions_column.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiPopover,
+  EuiTitle,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { LogEntryColumnContent } from './log_entry_column';
+import { euiStyled } from '../../../../../observability/public';
+
+interface LogEntryActionsColumnProps {
+  isHovered: boolean;
+  isMenuOpen: boolean;
+  openMenu: () => void;
+  closeMenu: () => void;
+  onViewDetails: () => void;
+}
+
+export const LogEntryActionsColumn: React.FC<LogEntryActionsColumnProps> = ({
+  isHovered,
+  isMenuOpen,
+  openMenu,
+  closeMenu,
+  onViewDetails,
+}) => {
+  const handleClickViewDetails = useCallback(() => {
+    closeMenu();
+    onViewDetails();
+  }, [closeMenu, onViewDetails]);
+
+  const label = i18n.translate('xpack.infra.logEntryItemView.logEntryActionsMenuToolTip', {
+    defaultMessage: 'View Details',
+  });
```

Comment:
Do we want to move this outside of the component so the translation is not performed on every render?